tests/test_helpers.py

```diff
@@ -860,6 +833,26 @@
 #      "List of all tags in citations collection:\n['nomonth', 'pdf']\nbuilding lists for all tags in the citation collection\nnomonth has been added/updated in reading_lists\npdf has been added/updated in reading_lists\n"),
 #     ]
 
+helper_map_bad = [
+ (["helper", "u_milestone", "--milestone_uuid", "sb_fir", "--projectum_id", "sb_firstprojectum",
+   "--due-date", "2023-01-01", "--notes", "do this", "do that", "--type", "mergedpr"],
+  "A uuid fragment and projectum id have been entered.\n"
+  "You may enter a milestone uuid or a projectum id but not both.\n"
+  "Enter a milestone uuid to update an existing milestone, or a projectum id to add a new milestone to that projectum.\n"
+  ),
+ (["helper", "u_milestone", "--due-date", "2023-01-01",
+   "--notes", "do this", "do that", "--type", "mergedpr"],
+  "No milestone uuid or projectum id was entered.\n"
+  "Enter a milestone uuid to update an existing milestone, or a projectum id to add a new milestone to that projectum.\n"
+  )]
```

Comment:
I think there are a  bunch more of these you can test.  Look above to see the blocks that Codecov show are not covered.  Any of those that result in an exception being thrown can be tested here.

You may have to put the type of error into the paramatrize options (i.e., pass in whether it is `RuntimeError` or `ValueError` or whatever as a variable as part of the test.